conf/application.conf

```diff
@@ -0,0 +1,400 @@
+    # The health check timeout will therefore be set between the first two, i.e. 50s to allow
+    # the timeout source to be identified
+    poll-timeout = 50 seconds
+    poll-count-between-updates = 20
+    poll-count-failures-to-alert = 3
+    poll-count-between-pager-duty-alerts = 20
+    # in order to make sure that only one health check is running across the different instances
+    # of this application in each environment a lock needs to be acquired before the health check
+    # will start. This parameter controls how long this lock is valid for and when the health check
+    # will try to acquire the lock again
+    lock-duration = 65 minutes
+  }
+
+}
+
+
+nino-logging.enabled = true
+
+email-validation {
+  # the maximum total length of an email address
+  max-total-length = 254
+
+  # the maximum length of the local part of email address
+  max-local-length = 64
+
+  # the maximum length of the domain part of email address
+  max-domain-length = 252
+}
+
+feature-toggles {
+  create-account-json-validation {
+    enabled = true
+  }
+
+  log-account-creation-json {
+    enabled = false
+  }
+}
+
+play.ws.ssl {
+  # Default platform timeout is 60s which is fine as-is
+  # timeout.request = 60s
+
+  # Handles the server root CA cert list; used by the client to verify the server
+  trustManager = {
+    stores = [
+      # Play seems to be loading default truststore without requiring any explicit config or password...
+      #{
+      # path: ${java.home}/lib/security/cacerts, # Default trust store
+      # password = ${microservice.services.nsi.server.base64TruststorePassword}, # Default trust store password
+      # }
+      {
+        data = ${microservice.services.nsi.server.base64Certificate}
+      }
+
+      //      {
+      //        data = ${microservice.services.nsi.server.phase2.set2.base64Certificate}
+      //      }
+      //      {
+      //        data = ${microservice.services.nsi.server.phase2.set1.base64Certificate}
+      //      }
+    ]
+  }
+
+# Handles the client cert (PrivateKeyEntry consisting of 3 chained certs) as
+# supplied by NS&I; used by the server to verify the client
+keyManager = {
+  stores = [
+    {
+      type = ${microservice.services.nsi.client.base64KeystoreType}
+      password = ${microservice.services.nsi.client.base64KeystorePassword}
+      data = ${microservice.services.nsi.client.base64Keystore}
+    }
+
+    //      {
+    //        type = ${microservice.services.nsi.client.base64KeystoreType}
+    //        password = ${microservice.services.nsi.client.phase2.set2.base64KeystorePassword}
+    //        data = ${microservice.services.nsi.client.phase2.set2.base64Keystore}
+    //      }
+    //      {
+    //        type = ${microservice.services.nsi.client.base64KeystoreType}
+    //        password = ${microservice.services.nsi.client.phase2.set1.base64KeystorePassword}
+    //        data = ${microservice.services.nsi.client.phase2.set1.base64Keystore}
+    //      }
+  ]
```

Comment:
you can get rid of comments above and above

Reply:
comments removed!!